src/main/java/seedu/revision/ui/StartQuizWindow.java

```diff
@@ -158,7 +152,25 @@ protected CommandResult executeCommand(String commandText) throws CommandExcepti
             CommandResult commandResult = logic.execute(commandText, currentAnswerable);
             if (commandResult.isCorrect()) {
                 totalScore++;
-                score++;
+                raw.put("total", raw.get("total") + 1);
+                switch (currentAnswerable.getDifficulty().difficulty) {
+                case "1":
+                    raw.put("difficulty 1", raw.get("difficulty 1") + 1);
+                    break;
+                case "2":
+                    raw.put("difficulty 2", raw.get("difficulty 2") + 1);
+                    break;
+                case "3":
+                    raw.put("difficulty 3", raw.get("difficulty 3") + 1);
+                    break;
+                default:
+                    assert false : currentAnswerable.getDifficulty().difficulty;
+                }
+
+                for (Enumeration i = raw.keys(); i.hasMoreElements();) {
+                    results.put((String) i.nextElement(), String.format((raw.get(i.nextElement())) + "/"
+                            + getSizeOfCurrentLevel(currentAnswerable)));
+                }
```

Comment:
Maybe can handle this inside of the statistics class using  statistics.update(currentAnswerable)

Reply:
Ok!